k8s/README.md

```diff
@@ -0,0 +1,400 @@
+# linkerd on Kubernetes #
+
+This doc explains how to get Gob's Microservice up and running 
+
+This example should work in any kubernetes cluster.  If you don't have one at
+your disposal, though, we can walk you through setting one up:
+
```

Comment:
I think we should also have a summary of what this README is going to cover including:

* an overview of the topology we'll set up (each of the GOB services running in a k8s pod with a a sidecar linkerd in the same pod, as well as a single global namerd)
* deploying the application
* staging with per-request routing
* canary with fractional traffic shifting
* fallback